code/stable/glassbr/src/cpp/designLog.txt

```diff
@@ -19,6 +19,7 @@ Successfully matched Real with Double.
 Successfully matched Boolean with Boolean.
 Successfully matched Rational with Double.
 Successfully matched Integer with Integer.
+Successfully matched Function (String :| [Real,Real]) Real with Func [String] Double.
```

Comment:
Why is this change happening here? Is something broken in `master`? @JacquesCarette @balacij

Reply:
Yep, irrelevant to this work. It seems to have happened earlier but never caught because `diff` checking ignores `.txt` files, so `stabilize` was never executed on something that changed.

Reply:
#3411

Reply:
So this can be ignored because it is being fixed elsewhere.

Reply:
Oops! Forgot to re-review